### PR TITLE
Transform some files in learn/css to markdown

### PR DESCRIPTION
**Pull request by jasonren0403**
Transform:
* learn/css/howto
* learn/css/first_steps/*

to markdown and perform a little link fix

### CODE REVIEW

**Review by yin1999 — CHANGES_REQUESTED** [review]
Hi @jasonren0403, please spread markdown convertion commit into two commits:

1. the first is used for renaming file (from `.html` to `.md`, using command `git mv`)
2. the second is used for markdown convertion with flow fixes, content sync, etc.

You can refer this pr: https://github.com/mdn/translated-content/pull/7395/commits

**Comment by jasonren0403** [comment]
sorry, please wait for another push for content fixes...I accidentallly force-pushed them by mistake 😭

**Comment by yin1999** [comment]
> sorry, please wait for another push for content fixes...I accidentallly force-pushed them by mistake 😭

Feel free to do that, and it's better to convert to draft (I'll do it for this time)

**Comment by jasonren0403** [comment]
> > sorry, please wait for another push for content fixes...I accidentallly force-pushed them by mistake 😭
> 
> Feel free to do that, and it's better to convert to draft (I'll do it for this time)

reorganize complete, ready to merge 😄

File: files/zh-cn/learn/css/first_steps/getting_started/index.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+        <li>项目 2</li>
+        <li>项目 <em>三</em></li>
+    </ul>
+
+</body>
+
+</html>
+```
+
+> **注意：** 假设你电脑操作环境不方便创建文件运行代码，别担心，可以用我们下方给出的在线实时代码编辑器。</strong></p></div>
```
Comment (yin1999) [inline files/zh-cn/learn/css/first_steps/getting_started/index.md:65]: the prefix for three note cards: <https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E5%A4%87%E6%B3%A8%E3%80%81%E8%AD%A6%E5%91%8A%E5%92%8C%E6%A0%87%E6%B3%A8>

```suggestion
> **备注：** 假设你电脑操作环境不方便创建文件运行代码，别担心，可以用我们下方给出的在线实时代码编辑器。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
Please follow the document to fix all of them

**Review by yin1999 — APPROVED** [review]
Thank you

**Pull request merged**